docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,98 @@
+# Networking Introduction
+
+In this chapter, you will find a very short introduction to various networking concepts. 
+These concepts are important to understanding when to use QUIC.
+
+## 1. TCP/IP and UDP Comparison
+
+Lets define some transport guarantees and their properties:
+
+| Transport Guarantees | Packet Loss [(1)][1]  | Packet Duplication [(2)][2] | Packet Order | Packet Delivery |
+| :-------------:   | :-------------: | :-------------: | :-------------: | :-------------:
+|   **Unreliable**  |       Any       |      Yes        |     No          |   No
+|   **Reliable**    |       No        |      No         |     Ordered     |   Yes
+
+Now, lets compare TCP, UDP, and QUIC.
+
+| Feature |  TCP  | UDP | QUIC
+| :-------------: | :-------------: | :-------------: | :-------------: |
+|  [Connection-Oriented][6]           |       Yes         | No                       | Yes
+|  Transport Guarantees              | Reliable Ordered  | Unreliable               | Reliable Ordered and Unreliable 
+|  Packet Transfer                    | [Stream-based][4] | Message based            | Message based and/or Stream based
+|  Automatic [fragmentation][8]       | Yes               | Yes, ip-fragmentation    | Yes
+|  Header Size                        |  20 bytes         | 8 bytes                  |  16 bytes  
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes  | No                       |  Yes, and user controlled                                          
+|  Based On | [IP][3]                | [IP][3]            |  UDP
```

Comment:
Let's merge the transport guarantees/packet transfer rows here. I think we should probably just skip the fragmentation row, it doesn't seem very important. The header size for QUIC is not 16 bytes, it really depends on the length of the chosen connection IDs. In the control flow/congestion avoidance row, I believe the Yes/No distinction between TCP and UDP is pretty unhelpful, and "user controlled" seems inaccurate. I think you mean here that most QUIC implementations will run in userspace compared to TCP running control flow/congestion stuff in kernelspace, but there might be kernel implementations of QUIC in the future, and "user" is not a very clear representation of "userspace".

Reply:
Let's merge the transport guarantees/packet transfer rows here.  I am not sure how this should be done. Do we want to keep:  Packet Loss/Packet Duplication/Packet Order/Packet Delivery? Or reduce it unreliable/reliable? and if so do we need to we still want to mention the first mentioned points?

Reply:
Reducing it to unreliable and reliable is probably the right way forward.